### ISSUE TITLE
fitparamodel: remove ObjFun option

### ISSUE THREAD

**Issue by stestoll** [luisfabib/DeerAnalysis2#75]
Currently, one can set the `'ObjFun'` option to select between `'ssr'` (sum of squared residuals), `'chi2'` (chi-square), '`chi2red'` (reduced chi-square).

For a single signal, these quantities differ only by constant factors, so have no effect on minimization. For global fitting, `'ssr'` does not include the noise, but the other do. Noise levels should always be included in global fits when available..

Bottom line: We should remove this option, since it doesn't affect minimization.
